### DISCUSSION TITLE
Releases

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/GoOdf#73] [General]
Is there a possibility to know the changes of the updates like with GrandOrgue ?

**Comment by larspalo** [comment]
As I answered on https://github.com/larspalo/GOODF/discussions/31#discussioncomment-6282084 there's no changelog file, but it's fairly easy to go to the release page and click on the "Compare" button to get all changes (commits) against the previous version.

![ChangesStep1](https://github.com/larspalo/GOODF/assets/6173941/aad4a25b-1bf6-4c99-8d09-2b2adf7e115d)
![ChangesStep2](https://github.com/larspalo/GOODF/assets/6173941/c098224a-27c4-4820-a839-06db08f39e21)
![ChangesStep3](https://github.com/larspalo/GOODF/assets/6173941/3e778e06-2801-4cca-840e-abf6ab8fbe61)

The main new features are:
- import of voicing data from a .cmb file is now possible
- tooltips can be enabled/shown
- Recently used files are shown and can be quickly accessed. This feature paves the way for the possibility to save more user settings, which will be implemented in the future

**Reply by Jean-Gabriel1982** [reply]
Thank you very much and sorry for the double post ..

**Reply by larspalo** [reply]
No problem, don't worry! GOODF is still very much under development so naturally many things are not yet streamlined and can change radically on short (or no) notice. I'm grateful to everyone that help testing the software!
